README.md

```diff
@@ -219,8 +219,8 @@ export default [
 
 ## Editor Integrations
 
-**Visual Studio Code**  
-Install [dbaeumer.vscode-eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).  
+**Visual Studio Code**
+Install [dbaeumer.vscode-eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
```

Comment:
Could you please revert the unrelated change? Or add `\` to the end of the line?

```suggestion
**Visual Studio Code**\
Install [dbaeumer.vscode-eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).\
```

Reply:
This was done by pnpm run update 🤷

Reply:
Hmm. Maybe it's because `.editorconfig` was added recently? I don't use the EditorConfig plugin so I'm not sure 😓

Reply:
I've changed it to use the `/` notation, so could you rebase it? #1215